sdcm/utils/benchmarks.py

```diff
@@ -12,13 +12,55 @@
 # Copyright (c) 2021 ScyllaDB
 import logging
 import re
+from dataclasses import dataclass, field
+from pathlib import Path
 
 from sdcm.remote import RemoteCmdRunnerBase, shell_script_cmd
+from sdcm.utils.common import ParallelObject
 from sdcm.utils.metaclasses import Singleton
 
 LOGGER = logging.getLogger(__name__)
 
 
+@dataclass
+class ScyllaNodeBenchmarkSysbenchResult:
+    # pylint:disable=too-many-instance-attributes
+    cmd_output: str = field(repr=False)
+    events_per_second: float = field(init=False)
+    latency_min: float = field(init=False)
+    latency_max: float = field(init=False)
+    latency_avg: float = field(init=False)
+    latency_95: float = field(init=False)
+    thread_fairness_avg: float = field(init=False)
+    thread_fairness_stdev: float = field(init=False)
+    thread_fairness_time_avg: float = field(init=False)
+    thread_fairness_time_stdev: float = field(init=False)
+
+    def __post_init__(self):
+        eps_regex = re.compile(
+            r"events per second:\s+(?P<eps>[\d.]+)([\s\w\d.:/()]+)"
+            r"min:\s+(?P<latency_min>[\d.]+)([\s\w\d.:/()]+)"
+            r"avg:\s+(?P<latency_avg>[\d.]+)([\s\w\d.:/()]+)"
+            r"max:\s+(?P<latency_max>[\d.]+)([\s\w\d.:/()]+)"
+            r"95th percentile:\s+(?P<latency_95>[\d.]+)([\s\w\d.:/()]+)"
+            r"events\s+\(avg/stddev\):\s+(?P<thread_avg>[\d.]+)/(?P<thread_stdev>[\d.]+)([\s\w\d.:/()]+)"
+            r"execution time \(avg/stddev\):\s+(?P<time_avg>[\d.]+)/(?P<time_stdev>[\d.]+)"
+        )
+        LOGGER.info("Running regex on:")
+        LOGGER.info(self.cmd_output)
+        with Path("sysbench_output").open(mode="w") as outfile:
+            outfile.write(self.cmd_output)
+        # self.events_per_second = eps_regex.search(self.cmd_output).groupdict()["eps"]
+        # self.latency_min = eps_regex.search(self.cmd_output).groupdict()["latency_min"]
+        # self.latency_max = eps_regex.search(self.cmd_output).groupdict()["latency_max"]
+        # self.latency_avg = eps_regex.search(self.cmd_output).groupdict()["latency_avg"]
+        # self.latency_95 = eps_regex.search(self.cmd_output).groupdict()["latency_95"]
+        # self.thread_fairness_avg = eps_regex.search(self.cmd_output).groupdict()["thread_avg"]
+        # self.thread_fairness_stdev = eps_regex.search(self.cmd_output).groupdict()["thread_stdev"]
+        # self.thread_fairness_time_avg = eps_regex.search(self.cmd_output).groupdict()["time_avg"]
+        # self.thread_fairness_time_stdev = eps_regex.search(self.cmd_output).groupdict()["time_stdev"]
```

Comment:
Any helpful comment for this block of 'commented out' code?

Reply:
Fixed with rebase (the commented out code was not present in the latter PR commits, now it won't show up in the earlier ones either)